lib/tasks/import_remediations.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+desc 'Update rule remediation availability by querying the Remdiations API'
+task import_remediations: :environment do
+  start_time = Time.now.utc
+  Rails.logger.info("Starting import_remediations job at #{start_time}")
+  RemediationsAPI.new(
+    Account.find_by(account_number: ENV['JOBS_ACCOUNT_NUMBER'])
+  ).import_remediations
+  end_time = Time.now.utc
+  duration = end_time - start_time
+  Rails.logger.info("Finishing import_remediations job at #{end_time} "\
+                    "and last #{duration} seconds ")
```

Comment:
Not a huge deal, but this logs to the rails logger. When running a rake task, I expect logs to output to STDOUT from where I'm running the task.

Reply:
I thought RAILS_LOG_TO_STDOUT would also force this to print it out, but it doesn't - I'm changing this.